packages/pipeline-editor/src/PipelineExportDialog.tsx

```diff
@@ -49,15 +49,26 @@ interface IProps {
   runtimeData: IRuntimeData;
   runtimeTypeInfo: IRuntimeType[];
   pipelineType?: string;
+  exportName: string;
 }
 
 export const PipelineExportDialog: React.FC<IProps> = ({
   runtimeData,
   runtimeTypeInfo,
-  pipelineType
+  pipelineType,
+  exportName
 }) => {
   return (
     <form className="elyra-dialog-form">
+      <label htmlFor="export_name">Export Name:</label>
+      <br />
+      <input
+        type="text"
+        id="export_name"
+        name="export_name"
+        defaultValue={exportName}
+        data-form-required
+      />
```

Comment:
Can you add one or two more break lines after this input element, so it gives some spacing between that and the next element below?

Reply:
Made the changes, and added two break lines.